subcommands/devices/config_updates.go

```diff
@@ -26,7 +26,12 @@ currently configured and reporting.`,
 
   # Set the docker apps and the tag:
   fioctl devices config updates <device> --apps shellhttpd --tag master
-`,
+
+  # Set the docker apps to none:
+  fioctl devices config updates <device> --apps ,
+
+  # Set the docker apps to inherit from parent:
+  fioctl devices config updates <device> --apps -`,
```

Comment:
explain what "parent" and "none" mean here also

Reply:
I would state explicitly here that we are inheriting from the group config.

```suggestion
# Set the docker apps to inherit from a group config:
```

We might be more verbose (like below), but I think that is apparent from the `fioctl config updates` help.
```suggestion
# Set the docker apps to inherit from a group config (if set there) or a system default (otherwise):
```

Reply:
Once I wrote this I understood we don't yet have a way to restore the device tag to inherit from a group config or system default.
Do we need one?

@doanac @mike-scott ^^^

Reply:
probably should do it also

Reply:
note: Our UI refers to this as using "preset-apps" and "preset-tag". Probably should make this consistent